generators/server/templates/src/main/resources/config/_application.yml

```diff
@@ -123,7 +123,7 @@ jhipster:
         corePoolSize: 2
         maxPoolSize: 50
         queueCapacity: 10000
-    #cors: #By default CORS are not enabled. Uncomment to enable.
+    #cors: #By default CORS is disabled. Uncomment to enable.
```

Comment:
Yes I think it's better, I'll have a look and do the library part if that's good for you